.github/workflows/release-branch.yaml

```diff
@@ -0,0 +1,80 @@
+name: Create OCM Release Branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_branch:
+        description: 'Branch to cut to, must match regex releases/v0.[0-9]+'
+        required: true
+      source_branch:
+        description: 'Branch to cut from (default main)'
+        required: true
+        default: 'main'
+
```

Comment:
Needs a top level `permissions:` with Read and then the individual jobs need to specify their write request.

Reply:
I believe it doesnt because default permission is read and we use app tokens

Reply:
if the scanner complains i will triage it to death!